app/services/stats/management_information/concerns/journey_queryable.rb

```diff
@@ -100,6 +100,7 @@ def journeys_query
               round(c.total + c.vat_amount, 4) as claim_total,
               c.last_submitted_at at time zone 'utc' at time zone 'Europe/London' as last_submitted_at,
               c.original_submission_date at time zone 'utc' at time zone 'Europe/London' as originally_submitted_at,
```

Comment:
These names were already applied as part of the previous refactoring work. They were named after what their csv header column is to avoid confusion in that context. I can't say I am keen on naming them after the timezone either.

Reply:
Yes, I missed it in the previous refactoring work.

I didn't realise that they were named after the csv header columns. This isn't a blocker bit I can foresee that the slight language change between `original_submission_date` and `originally_submitted_at`, for what is essentially the same data in different time zones, could cause confusion.